gdcmodels/esutils.py

```diff
@@ -0,0 +1,64 @@
+import logging
+import urllib3
+
+from time import sleep
+
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionTimeout
+
+UPDATE_INTERVAL = 10
+
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def get_elasticsearch(host, port=9200, user='', password='', use_ssl=True):
+    """Create an Elasticsearch client according to the given CLI args.
+
+    Args:
+        host: Elasticsearch host
+        port: Elasticsearch port
+        user: Elasticsearch username
+        password: Elasticsearch password
+        use_ssl: Turn on/off SSL
+
+    Returns:
+
+    """
+    return Elasticsearch(
+        hosts=[{"host": host, "port": port}],
+        http_auth=(user, password),
+        use_ssl=use_ssl,
+        verify_certs=False,
+        timeout=60,
+    )
+
+
+def force_merge_elasticsearch_indices(es, index, max_num_segments=1):
+    """force merge segments in elasticsearch indices
+
+    Args:
+        es: Elasticsearch object, Elasticsearch low-level client
+        index: list of index names to merge
+        max_num_segments: The number of segments the index should be merged into
+
+    Returns:
+
+    """
+    if not index or not isinstance(index, list):
```

Comment:
The [docs](https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch.client.IndicesClient.forcemerge) says the parameter `index` is a comma separated list of index names :thinking:

Reply:
This is an unfortunately common problem with the documentation for the Elasticsearch Python client library. The _REST API_ expects a comma-separated string, so the documentation asks for a string, but really, the Python library will take an actual list. The list will be ["escaped"](https://github.com/elastic/elasticsearch-py/blob/7.7.1/elasticsearch/client/utils.py#L12-L20) into a comma-separated string before it is sent to the REST API.

Given that mutation indexer will need to force-merge several indices, I think it would be helpful if this function could accept a list like it did before @qiaouchicago's most recent commit. We should be able to pass that list directly into the client library. We could also make it accept either a list or a string, but our immediate use cases will all involve multiple indices, so the list will be more useful for us.